modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs using the YAML view
+
+.Prerequisites
+
+* You must have administrator privileges.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Administration* -> *Cluster Settings*.
```

Comment:
Can you please provide the link for bonus convenience?

Reply:
The link? May I know why? I think the users will know how to navigate up to here if they have followed the steps in the procedure.

Reply:
You're right.

Reply:
Fixed.